source/MRViewer/MRViewer.h

```diff
@@ -63,6 +63,7 @@ struct LaunchParams
     bool isAnimating{ false }; // if true - calls render without system events
     int animationMaxFps{ 30 }; // max fps if animating
     bool unloadPluginsAtEnd{ false }; // unload all extended libraries right before program exit
+    bool initializeSpaceMouseHandler{ true }; // whether to initialize SpaceMouse handler together with Viewer
```

Comment:
Mention that the handler can be initialized manually.

Reply:
Option removed